eng/Versions.props

```
@@ -3,6 +3,7 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>17.8.0</VersionPrefix>
+    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

I know it's ugly but putting this on the _same line_ helps get merge conflicts into good shape for the future.
```suggestion
    <VersionPrefix>17.8.0</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```